### ISSUE TITLE
report many error on pyside6

### ISSUE THREAD

**Issue by zhuangzhuang** [microsoft/pyright#1329]
**Describe the bug**
![捕获](https://user-images.githubusercontent.com/521058/103332408-c5e2d000-4aa4-11eb-9892-14f20828049a.PNG)

**To Reproduce**
1. install pyside6
2. use this code https://doc.qt.io/qtforpython-6/quickstart.html

**Expected behavior**

**Screenshots or Code**
If applicable, add screenshots or the text of the code (surrounded by triple back ticks) to help explain your problem.
```python
import sys
import random
from PySide6 import QtConcurrent, QtCore, QtWidgets


class MyWidget(QtWidgets.QWidget):
    def __init__(self):
        super().__init__()

        self.hello = ["Hallo Welt", "Hei maailma", "Hola Mundo", "Привет мир"]

        self.button = QtWidgets.QPushButton("Click me!")
        self.text = QtWidgets.QLabel("Hello World",
                                     alignment=QtCore.Qt.AlignCenter)

        self.layout = QtWidgets.QVBoxLayout()
        self.layout.addWidget(self.text)
        self.layout.addWidget(self.button)
        self.setLayout(self.layout)

        self.button.clicked.connect(self.magic)

    @QtCore.Slot()
    def magic(self):
        self.text.setText(random.choice(self.hello))


if __name__ == "__main__":
    app = QtWidgets.QApplication([])

    widget = MyWidget()
    widget.resize(800, 600)
    widget.show()

    sys.exit(app.exec_())

```

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool? Which version? You can find the version of the VS Code extension by clicking on the Pyright icon in the extensions panel.

**Additional context**

**Comment by erictraut** [comment]
The pyside6 library contains a "py.typed" marker, which means (according to PEP 561) that it provides its own type information either in the form of inline type annotations or packaged stubs. It looks like pyside6 uses a combination of these two approaches.

Unfortunately, there are many problems with the type information included in the library. It appears to be missing parameters and methods. It also conditionally imports symbols from an alternate `typing` module, which obscures types like `typing.overload` from the type checker.

If this is a library you care about, you should contact the maintainers and file bugs or PRs to fix the issues you find. [This documentation](https://github.com/microsoft/pyright/blob/master/docs/typed-libraries.md) provides detailed guidance for library authors and maintainers.

In any case, I think pyright is doing the right thing, so I'm going to close this issue.
